source/solvers/navier_stokes_base.cc

```diff
@@ -66,12 +66,18 @@ NavierStokesBase<dim, VectorType, DofsType>::NavierStokesBase(
       const FE_SimplexP<dim> pressure_fe(
         p_nsparam.fem_parameters.pressure_order);
       fe = std::make_shared<FESystem<dim>>(velocity_fe, dim, pressure_fe, 1);
-      velocity_mapping = std::make_shared<MappingFE<dim>>(velocity_fe);
-      pressure_mapping = std::make_shared<MappingFE<dim>>(pressure_fe);
+      mapping = std::make_shared<MappingFE<dim>>(velocity_fe);
+      mapping_q_all_cells =
+        std::make_shared<MappingFE<dim>>(velocity_fe); // dummy
+      mapping_no_q_all_cells =
+        std::make_shared<MappingFE<dim>>(velocity_fe); // dummy
```

Comment:
I don't understand why you have so many different mappings? Why not a single mapping for velocity and that's it?